### ISSUE TITLE
Room Editor - Pasting copied instance not respecting grid

### ISSUE THREAD

**Issue by iampremo** [YoYoGames/GameMaker-Bugs#900]
GM-7468

"Description: I have 3 grey blocks in my room editor. The sprites for these are 256x256px. I place a grey block in the room editor and resize it there to 64x64. Now, sometimes when I copy and paste the grey block using CTRL+C and CTRL+V it gets placed in a way that makes it impossible to allign with the other blocks. I know you can press ctrl and manually move it pixel by pixel but this is unproductive and may not neccessarily end up with properly aligned objects.

Basically it would be nice if a copy and paste respected the grid size chosen."

Steps
- import the attached project
- open the room
- click on of the grey boxes
- press copy (ctrl+c)
- press paste (ctrl+v) notice there might be a slight mis-alignment
- keep pasting and notice the instances do not match the grid and sometimes the mis-alignment grows
this occurs if snap to grid is selected or not

**Comment by phablix** [comment]
I run into this issue too

![Image](https://github.com/user-attachments/assets/e67a8256-d7c2-4a74-9e84-4809805e7a02)

Sometimes when copy pasting scaled nine slice stuff there is a chance of rounding errors (about 1/10 times) even if snap to grid is enabled.
That mess with pixel perfect stuff. And the solution of flooring manually in code is kinda messy.

It can also happen to position and not only scale but it's more rare.
